src/skins/vector/views/molecules/ServerConfig.js

```diff
@@ -20,37 +20,142 @@ var React = require('react');
 var Modal = require('matrix-react-sdk/lib/Modal');
```

Comment:
Also, how come PasswordLogin & CasLogin live in react-sdk but this lives in vector?